src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -16,26 +18,26 @@ public class Exercise {
 
     // 2. There are currently some upper case characters in the URL. Using an appropriate string method on the fixedUrl member above,
     // set the value of lowerCasedUrl.
-    public String lowerCasedUrl = "";
+    public String lowerCasedUrl = fixedUrl.toLowerCase();
 
 
     // 3. There is still white space on both ends of the URL! Use the appropriate string method to trim that white space
     // and set the value of the url member below
-    public String url = "";
+    public String url = lowerCasedUrl.replaceAll("\\s+","");
```

Comment:
Good if you want to remove all whitespace in a string. `trim()` is a quick/easy way to remove trailing whitespaces.